core/templates/components/forms/schema-based-editors/schema-based-unicode-editor.component.ts

```diff
@@ -16,7 +16,7 @@
  * @fileoverview Component for a schema-based editor for unicode strings.
  */
 
-require('third-party-imports/ui-codemirror.import.ts');
+import '../../../third-party-imports/ui-codemirror.import';
```

Comment:
hacky work around to get both the angular compiler and webpack build to not complain.

Reply:
Actually, I think this is ok? At least it seems reasonable to me in terms of providing the correct relative path, not sure what the better alternative is (maybe an absolute path from root would be nicer).